test-jest/chart/Sankey.spec.tsx

```diff
@@ -127,30 +126,30 @@ describe('<Sankey />', () => {
     ],
   };
 
-  const wrapper = render(<Sankey width={1000} height={500} data={data} />);
-
   it('renders 48 nodes in simple SankeyChart', () => {
-    expect(wrapper.find('.recharts-sankey-node').length).to.equal(48);
+    const { container } = render(<Sankey width={1000} height={500} data={data} />);
+
+    expect(container.querySelectorAll('.recharts-sankey-node')).toHaveLength(48);
   });
 
   it('renders 68 links in simple SankeyChart', () => {
-    expect(wrapper.find('.recharts-sankey-link').length).to.equal(68);
+    const { container } = render(<Sankey width={1000} height={500} data={data} />);
+
+    expect(container.querySelectorAll('.recharts-sankey-link')).toHaveLength(68);
   });
 
   it('re-renders links and nodes when data changes', () => {
-    const wrapper = mount(<Sankey width={1000} height={500} data={data} />);
-    expect(wrapper.render().find('.recharts-sankey-node').length).to.equal(48);
-    expect(wrapper.render().find('.recharts-sankey-link').length).to.equal(68);
+    const { container } = render(<Sankey width={1000} height={500} data={data} />);
+
+    expect(container.querySelectorAll('.recharts-sankey-node')).toHaveLength(48);
+    expect(container.querySelectorAll('.recharts-sankey-link')).toHaveLength(68);
+
+    data.nodes.push({ name: 'New Node' });
```

Comment:
Are you aware of this approach? https://testing-library.com/docs/example-update-props/

I do actually like yours better, though.

Reply:
Yes, I learned it from your PR.
I agree with you. I think that changing the value rather than reforcing rerender is more proper for this test.